src/model/uniprotkb/sections/SequenceConverter.ts

```diff
@@ -142,6 +142,28 @@ export const convertSequence = (data: UniProtkbAPIModel) => {
       return sequenceFeatures.includes(feature.type);
     });
     sequenceData.featuresData = features;
+    // Add VAR_SEQ to corresponding isoforms
+    if (features && sequenceData.alternativeProducts) {
+      const varSeqs = features.filter(
+        feature => feature.type === FeatureType.VAR_SEQ
+      );
+      sequenceData.alternativeProducts.isoforms = sequenceData.alternativeProducts.isoforms.map(
+        isoform => {
+          const varSeqsToAdd: FeatureData = [];
+          if (isoform.sequenceIds) {
+            isoform.sequenceIds.forEach(sequenceId => {
+              const varSeqToAdd = varSeqs.find(
```

Comment:
Could create a mapping before the map at L150 which goes from `varSeq.featureId` --> `varSeq` to eliminate the need for `finds`.

Reply:
It's a relatively small map. I'm not sure it would add anything.